### DISCUSSION TITLE
0 mods problem

### DISCUSSION THREAD

**Discussion by prem0616** [thatskymod/Sky-CotL-Scripts#127] [General]
in pc i installed all those and kept the files and after that when i open the game in the mod menu their are no mods it showing 0 mods help me fix it.
![Uploading sky prob.png…]()

**Comment by thatskymod** [comment]
Hello,

It looks like your screenshot didn’t finish uploading, but I can offer some general guidance.

You might want to try the [ThatSkyMod](https://github.com/XeTrinityz/ThatSkyApp/) installer by @XeTrinityz, which simplifies downloading and installing mods. The installer automatically places everything in the correct location. Using this tool should resolve your issue with the 0 mods showing up.

Let me know if this helps!
